minerl/data/pipeline/publish.py

```diff
@@ -349,8 +368,12 @@ def render_data(output_root, recording_dir, experiment_folder, black_list, lineN
                         continue
                 raise e
 
-            # Don't release ones with 1024 reward (they are bad streams) and other smoke-tests
-            if 'Survival' not in environment.name and not isinstance(environment, Obfuscated):
+            if 'Caves' in environment.name:
+                # Without this exception, Caves is autoblacklisted because reward == 0.0.
```

Comment:
If minerl prefers, we can just assign 100 reward to snowball throw or something...

Reply:
Ha yeah, that would work in a pinch if we need that. The long-term solution (we don't want to imply that our environments come with actual reward fns or something I think) needs to be something more general though (i.e. EnvSpec.auto_blacklist).